Tweak voidsuits so they can only be unequipped by click-and-drag

MuckerMayhem opened the pull request:
🆑 Mucker
tweak: Voidsuits can now only be dequipped by click-dragging them into an open hand.
/🆑

I think everyone's come across a time when they accidentally pulled off their voidsuit while in space, or a hazardous area, due to a very unfortunate miss-click. This aims to stop that from happening.  Voidsuits can now only be taken off by clicking and dragging them into an open hand.

SierraKomodo commented on code/modules/clothing/spacesuits/void/void.dm:
```diff
@@ -307,6 +307,11 @@ else if(##equipment_var) {\
 	..()
 
+/obj/item/clothing/suit/space/void/attack_hand(mob/user as mob)
+	if (src.loc == user)
```
```suggestion
	if (loc == user)
```

MuckerMayhem committed:
remove ability to click to unequip voidsuits

Voidsuits can now only be unequipped by clicking and dragging them into an open hand,
